urllib3/util/ssl_.py

```diff
@@ -17,6 +18,23 @@
 
 import errno
 import warnings
+import hmac
+
+if hasattr(hmac, 'compare_digest'):
+    _const_compare_digest = hmac.compare_digest
+else:
+    def _const_compare_digest(a, b):
+        """
+        Compare two digests of equal length in constant time.
+
+        Returns True if the digests match, and False otherwise.
+        """
+        result = abs(len(a) - len(b))
+        for l, r in six.moves.zip(bytearray(a), bytearray(b)):
```

Comment:
You're really not getting anything from six here. :) `zip(...)` will be identical for our purposes.